sentry-android-navigation/src/main/java/io/sentry/android/navigation/SentryNavigationListener.kt

```diff
@@ -90,6 +90,7 @@ class SentryNavigationListener @JvmOverloads constructor(
         hub.addBreadcrumb(breadcrumb, hint)
     }
 
+    // TODO should we start a new trace here if performance is disabled?
```

Comment:
Need help deciding whether this should create a new `PropagationContext` with random `traceId` and `spanId` or if we rather want to keep the previous trace going.